Jellyfin.Windows.Tray/TrayApplicationContext.cs

```diff
@@ -18,7 +18,9 @@ public class TrayApplicationContext : ApplicationContext
         private readonly string _jellyfinServiceName = "JellyfinServer";
         private readonly string _autostartKey = "JellyfinTray";
         private string _configFile;
+        private string _networkFile;
         private bool FirstRunDone = false;
+        private string NetworkAddress;
```

Comment:
```suggestion
        private string _networkAddress;
```

Reply:
Also it looks like it is only used locally so no need for a class property I guess

Reply:
Applied change, will fix the rest in other commit.